### PR TITLE
Internationalizing Strings

### PR DESCRIPTION
**Pull request by TheJYKoder**
### Contains

Brief description of what the PR does like "Fixes #12345"

### How to test

Brief description of how to test / confirm this PR before merging

### Outstanding before merging

If anything. You can use neat checkboxes! Feel free to delete if not needed

- [ ] Need to consider use case x
- [ ] Still have to adjust the wiki doc
- [ ] Will need translation work

### CODE REVIEW

**Comment by GooeyHub** [comment]
Can one of the admins please verify this patch?

File: engine/src/main/resources/assets/ui/enterTextPopup.ui [hunk 1]
```diff
@@ -18,7 +18,7 @@
                     "contents": [
                         {
                             "type": "UILabel",
-                            "text": "Enter name",
+                            "text":  "${engine:menu#player-name}" + ": ",
```
Comment (Cervator) [inline engine/src/main/resources/assets/ui/enterTextPopup.ui]: Be careful in JSON - you cannot necessarily use the same string building as in Java :-)

File: engine/src/main/resources/assets/ui/waitPopup.ui [hunk 2]
```diff
@@ -29,7 +29,7 @@
                         },
                         {
                             "type": "UILabel",
-                            "text": "Message",
+                            "text": " "${engine:menu#please-wait}"",
```
Comment (Cervator) [inline engine/src/main/resources/assets/ui/waitPopup.ui]: Some broken quotes here. Fixed during merge.

**Comment by Cervator** [comment]
Happy fun Git times tonight! :-)

I don't know what happened over in #2730, but you somehow managed to derail the source branch pretty bad :D

"TheJYKoder wants to merge 16 commits into MovingBlocks:develop from **unknown repository**"

Am expecting you had prepared the Javadoc first, but then did some odd branch switching that didn't work, and later overwrote it, removed the branch, or something, and then managed to get this put together in your `develop` branch alone, without the Javadoc?

Anyway, GitHub has your changes cached in an older closed PR so I'm going to steal it from there manually and credit you for it + approve the GCI task so we can get that out of the way.

You had a couple odd spots in the JSON here as well, but I fixed them as part of the merge :-)

**Comment by Cervator** [comment]
Oh, forgot some related notes on the JavaDoc: - all pretty trivial:

* end first sentence in javadoc with a . (parser looks for one - good practice)
* don't worry about trying to align regular line comments
* no need to inline comment stuff like `if (count < 1) {  // The count must be greater than or equal to 1`
* leave newlines between the end of methods and the javadoc for the next method

**Pull request merged**